README.md

```diff
@@ -30,7 +30,8 @@ You will need to have pre-installed:
 
 - [Make](https://man7.org/linux/man-pages/man1/make.1.html)
 
-## Prerequisites
+The rest of the prerequisites are system-dependent:
```

Comment:
Does this sentence suppose to be empty after the column?

Reply:
You mean the colon? If so, yes, the next paragraph is "linux"